web/src/pages/people/index.tsx

```diff
@@ -27,14 +26,12 @@ export default function PeoplePage() {
         description={
           isAdmin
             ? "View and manage portal users, including adding via invitation or upload"
-            : isApprovedStaffResearcher
-              ? "View users in your projects"
-              : isIAO
-                ? "View users in your projects or invite a collaborator"
-                : "You do not have permission to view this page"
+            : isIAO
+              ? "View users in your projects or invite a collaborator"
+              : "You do not have permission to view this page"
         }
       />
-      {isAdmin ? <AdminView /> : isApprovedStaffResearcher && <ApprovedResearcherView isIAO={isIAO} />}
+      {isAdmin ? <AdminView /> : isIAO && <ApprovedResearcherView isIAO={isIAO} />}
```

Comment:
don't think we need to pass in `isIAO` since it has to be IAO to render the component